contracts/uniswap/UniswapV2PairBalancedAssetsTest.sol

```diff
@@ -0,0 +1,155 @@
+    require(pairAddress != address(0), "UniswapV2PairBalancedAssetsTest: pair address cannot be 0x0");
+    
+    protocolName = "Uniswap V2";
+    
+
+    // CHAINLINK FEED ETH / USD for token WETH
+    chainlinkFeedUSD[0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2] = 0x5f4eC3Df9cbd43714FE2740f5E3616155c5b8419;
+
+    // CHAINLINK FEED COMP / USD for token COMP
+    chainlinkFeedUSD[0xc00e94Cb662C3520282E6f5717214004A7f26888] = 0xdbd020CAeF83eFd542f4De03e3cF0C28A4428bd5;
+
+    // CHAINLINK FEED KNC / USD for token KNC
+    chainlinkFeedUSD[0xdd974D5C2e2928deA5F71b9825b8b646686BD200] = 0xf8fF43E991A81e6eC886a3D281A2C6cC19aE70Fc;
+
+    // CHAINLINK FEED LINK / USD for token LINK
+    chainlinkFeedUSD[0x514910771AF9Ca656af840dff83E8264EcF986CA] = 0x2c1d072e956AFFC0D435Cb7AC38EF18d24d9127c;
+    
+    // CHAINLINK FEED MANA / USD for token MANA
+    chainlinkFeedUSD[0x0F5D2fB29fb7d3CFeE444a200298f468908cC942] = 0x56a4857acbcfe3a66965c251628B1c9f1c408C19;
+
+    // CHAINLINK FEED USDP / USD for token USDP
+    chainlinkFeedUSD[0x8E870D67F660D95d5be530380D0eC0bd388289E1] = 0x09023c0DA49Aaf8fc3fA3ADF34C6A7016D38D5e3;
+
+    // CHAINLINK FEED TUSD / USD for token TUSD
+    chainlinkFeedUSD[0x0000000000085d4780B73119b644AE5ecd22b376] = 0xec746eCF986E2927Abd291a2A1716c940100f8Ba;
+
+    // CHAINLINK FEED USDC / USD for token USDC
+    chainlinkFeedUSD[0xA0b86991c6218b36c1d19D4a2e9Eb0cE3606eB48] = 0x8fFfFfd4AfB6115b954Bd326cbe7B4BA576818f6;
+
+    // CHAINLINK FEED USDT / USD for token USDT
+    chainlinkFeedUSD[0xdAC17F958D2ee523a2206206994597C13D831ec7] = 0x3E7d1eAB13ad0104d2750B8863b489D65364e32D;
+
+    // CHAINLINK FEED ZRX / USD for token ZRX
+    chainlinkFeedUSD[0xE41d2489571d322189246DaFA5ebDe1F4699F498] = 0x2885d15b8Af22648b98B122b22FDF4D2a56c6023;
+
+    // CHAINLINK FEED BAL / USD for token BAL
+    chainlinkFeedUSD[0xba100000625a3754423978a60c9317c58a424e3D] = 0xdF2917806E30300537aEB49A7663062F4d1F2b5F;
+
+    // CHAINLINK FEED YFI / USD for token YFI
+    chainlinkFeedUSD[0x0bc529c00C6401aEF6D220BE8C6Ea1667F6Ad93e] = 0xA027702dbb89fbd58938e4324ac03B58d812b0E1;
+
+    // CHAINLINK FEED GUSD / USD for token GUSD
+    chainlinkFeedUSD[0x056Fd409E1d7A124BD7017459dFEa2F387b6d5Cd] = 0xa89f5d2365ce98B3cD68012b6f503ab1416245Fc;
+
+    // CHAINLINK FEED UNI / USD for token UNI
+    chainlinkFeedUSD[0x1f9840a85d5aF5bf1D1762F925BDADdC4201F984] = 0x553303d460EE0afB37EdFf9bE42922D8FF63220e;
+
+    // CHAINLINK FEED BTC / USD for token WBTC
+    chainlinkFeedUSD[0x2260FAC5E5542a773Aa44fBCfeDf7C193bc2C599] = 0xF4030086522a5bEEa4988F8cA5B36dbC97BeE88c;
+
+    // CHAINLINK FEED AAVE / USD for token AAVE
+    chainlinkFeedUSD[0x7Fc66500c84A76Ad7e9c93437bFc5Ac33E2DDaE9] = 0x547a514d5e3769680Ce22B2361c10Ea13619e8a9;
+
+    // CHAINLINK FEED MATIC / USD for token MATIC
+    chainlinkFeedUSD[0x7D1AfA7B718fb893dB30A3aBc0Cfc608AaCfeBB0] = 0x7bAC85A8a13A4BcD8abb3eB7d6b4d632c5a57676;
```

Comment:
thanks :heart: , this is something i was looking for but couldn't find it. I'll update the PR ( and the others i have opened )

Reply:
@deemabot I've created a helper library for token to usd value using chainlink feed registry for ethereum mainnet. 
This would help with other tests that require token values.